packages/code/src/renderingPlugin.tsx

```diff
@@ -353,7 +353,12 @@ interface TableContentProps {
 }
 
 export function TableContent({ lines }: TableContentProps) {
-  const { highlightLines, showLineNumbers, darkMode } = useSyntaxContext();
+  const {
+    highlightLines,
+    showLineNumbers,
+    darkMode,
+    lineOffset,
+  } = useSyntaxContext();
```

Comment:
If you're adding the lineOffset to the context, it'll also need to be updated in the context's interface [here](https://github.com/mongodb/leafygreen-ui/blob/3c2fdcdeec868655db756813c9ba2dbf050598d5/packages/code/src/SyntaxContext.tsx#L4-L13)

I'm not 100% on if it needs to be in the context to work with this implementation - I don't think it's ever something that would be mutated on the fly, live on a page? But I also don't know if it's just a general convention, or something for a Storybook integration.